### ISSUE TITLE
Indicator status color contrast - "Statistics in progress"

### ISSUE THREAD

**Issue by brockfanning** [open-sdg/open-sdg#607]
The "Statistics in progress" status on goal pages is appearing as black text on a grey background. Apparently it passes the automated accessibility test, but to me it still seems hard to read. On the reporting status page we use white text, so I'm wondering if we should do the same here.

![statistics-in-progress-color-contrast](https://user-images.githubusercontent.com/1319083/78306567-b6345c00-7511-11ea-9e9e-cb995751dd65.png)

**Comment by brockfanning** [comment]
@SavvasStephanides @phillipgardiner I'm not sure if this needs to be done before version 1, but just bringing it to your attention, since it's related to stuff you've been looking at recently.

**Comment by SavvasStephanides** [comment]
Yes this looks like something that needs to be addressed. Not sure how it escaped.
